knowledge_repo/app/templates/markdown-base.html

```diff
@@ -89,6 +94,10 @@
   $(".btn-raw").on("click", function(){
     document.location.href = "/raw?markdown=" + encodeURI(post_path);
   })
+
+  $(".btn-webeditor").on("click", function(){
+    document.location.href = "/posteditor?path=" + encodeURI(post_path);
```

Comment:
That's a good point. But since there are other links with the same problem, let's resolve this in a different PR.